servicetalk-opentracing-zipkin-publisher/src/main/java/io/servicetalk/opentracing/zipkin/publisher/reporter/UdpReporter.java

```diff
@@ -147,15 +150,30 @@ public Builder executor(Executor executor) {
         }
 
         /**
-         * Enables wire-logging for udp packets sent.
+         * Enables wire-logging for UDP packets sent.
          * <p>
-         * All wire events will be logged at {@link Level#TRACE TRACE} level.
+         * All wire events will be logged at {@link LogLevel#TRACE TRACE} level.
          *
+         * @deprecated Use {@link #enableWireLogging(String, LogLevel, BooleanSupplier)} instead.
          * @param loggerName The name of the logger to log wire events.
          * @return {@code this}
          */
+        @Deprecated
         public Builder enableWireLogging(String loggerName) {
-            this.loggerName = loggerName;
+            return enableWireLogging(loggerName, TRACE, () -> false);
+        }
+
+        /**
+         * Enables wire-logging for UDP packets sent.
+         *
+         * @param loggerName The name of the logger to log wire events.
+         * @param logLevel The level to log at.
+         * @param logUserData {@code true} to include user data. {@code false} to exclude user data and log only
+         * network events.
+         * @return {@code this}
+         */
+        public Builder enableWireLogging(String loggerName, LogLevel logLevel, BooleanSupplier logUserData) {
```

Comment:
Wanted to remove all deprecated `enableWireLogging` but then found this inconsistency. Decided to align this builder for 0.38.0 release and we can remove them altogether in 0.39.0, see #1420.